gatsby_fp/src/components/fieldComponents/GenInfoTable.js

```diff
@@ -0,0 +1,121 @@
+                  <div className="table_wrapper">
+                    <div className="table_wrapper_inner">
+                      <table cellPadding="0" cellSpacing="0" border="0">
+                        <tbody>
+                          <tr>
+                            <td
+                              valign="top"
+                              dangerouslySetInnerHTML={{ __html: title }}
+                            ></td>
+                          </tr>
+
+                          <tr>
+                            <td valign="top">School: (Needs to be filled)</td>
+                          </tr>
+                          {building && (
+                            <tr>
+                              <td
+                                valign="top"
+                                dangerouslySetInnerHTML={{
+                                  __html: `Building: ${building}`,
+                                }}
+                              ></td>
+                            </tr>
+                          )}
+                          {room && (
+                            <tr>
+                              <td
+                                valign="top"
+                                dangerouslySetInnerHTML={{
+                                  __html: `Room: ${room}`,
+                                }}
+                              ></td>
+                            </tr>
+                          )}
+                          {phone && (
+                            <tr>
+                              <td
+                                valign="top"
+                                dangerouslySetInnerHTML={{
+                                  __html: `Phone: ${phone}`,
+                                }}
+                              ></td>
+                            </tr>
+                          )}
+                          {fax && (
+                            <tr>
+                              <td
+                                valign="top"
+                                dangerouslySetInnerHTML={{
+                                  __html: `Fax: ${fax}`,
+                                }}
+                              ></td>
+                            </tr>
+                          )}
+                          {username && (
+                            <tr>
+                              <td
+                                valign="top"
+                                dangerouslySetInnerHTML={{
+                                  __html: `Email: ${username}@stevens.edu`,
+                                }}
+                              ></td>
+                            </tr>
+                          )}
+                          {website && (
+                            <tr>
+                              <td
+                                valign="top"
+                                dangerouslySetInnerHTML={{
+                                  __html: `Website: ${website}`,
```

Comment:
It turns out that `website` always comes through as a URL, not as HTML. And if we look at an old profile such as https://web.stevens.edu/facultyprofile/?id=2007 we see how the word "Website" is actually the link text, the end user doesn't see "Website: http..." on the screen. So, we should rip out the dangerouslySetInnerHTML and just build an `<a>` tag where the `website` prop goes into the `href` attribute and the word "Website" is the innerHTML. Also, it should have `target="_blank"` just as the old profiles do.